File: Android/TUIKit/TUICallKit/tuicallkit/src/main/java/com/tencent/qcloud/tuikit/tuicallkit/extensions/recents/interfaces/ICallRecordItemListener.java
package com.tencent.qcloud.tuikit.tuicallkit.extensions.recents.interfaces;

import android.view.View;

import com.tencent.qcloud.tuikit.tuicallengine.TUICallDefine;

public interface ICallRecordItemListener {
    void onItemClick(View view, int viewType, TUICallDefine.CallRecords callRecords);

    void onItemDeleteClick(View view, int viewType, TUICallDefine.CallRecords callRecords);

    void onDetailViewClick(View view, TUICallDefine.CallRecords callRecords);
}

File: Android/TUIKit/TUICallKit/tuicallkit-kt/src/main/java/com/tencent/qcloud/tuikit/tuicallkit/extensions/recents/interfaces/ICallRecordItemListener.kt
package com.tencent.qcloud.tuikit.tuicallkit.extensions.recents.interfaces

import android.view.View
import com.tencent.qcloud.tuikit.tuicallengine.TUICallDefine.CallRecords

interface ICallRecordItemListener {
    fun onItemClick(view: View?, viewType: Int, callRecords: CallRecords?)
    fun onItemDeleteClick(view: View?, viewType: Int, callRecords: CallRecords?)
    fun onDetailViewClick(view: View?, callRecords: CallRecords?)
}